app/models/course/material.rb

```diff
@@ -0,0 +1,5 @@
+class Course::Material < ActiveRecord::Base
+  belongs_to :creator, inverse_of: :files, class_name: User.name
```

Comment:
I don't think the user should be linked to every file. ActiveRecord-Userstamp already creates the belongs_to association.